bin/build-darwin.sh

```diff
@@ -54,10 +57,10 @@ PATH="${DIR_THIRDPARTY}/bde-tools/bin:$PATH"
 
 if [ ! -e "${DIR_BUILD}/bde/.complete" ]; then
     pushd "${DIR_THIRDPARTY}/bde"
-    eval "$(bbs_build_env -u opt_64_cpp17 -b "${DIR_BUILD}/bde")"
-    bbs_build configure --prefix="${DIR_ROOT}"
-    bbs_build build --prefix="${DIR_ROOT}"
-    bbs_build --install_dir="/" --prefix="${DIR_ROOT}" install
+    eval "$(bbs_build_env -u opt_64_cpp17 -b "${DIR_BUILD}/bde" -i "${DIR_INSTALL}")"
```

Comment:
Did you try running the script? Specifically, I want to make sure that the `-i` option works as expected

Reply:
Yes---at least on Darwin:

    rm -rf build thirdparty ~/install-root
    export DIR_INSTALL=~/install-root
    ./bin/build-darwin.sh
    ls ~/install-root/lib
    
shows `libbal.a`, `libbbl.a`, `libntc.a` etc.